managed/models/service_model.go

```diff
@@ -99,6 +106,33 @@ func (s *Service) SetCustomLabels(m map[string]string) error {
 	return setLabels(m, &s.CustomLabels)
 }
 
+// ChangeStandardLabels changes standard labels and returns a list of columns which were updated.
+func (s *Service) ChangeStandardLabels(labels ServiceStandardLabelsParams) []string {
+	columns := []string{}
+
+	if labels.Cluster != nil {
+		columns = append(columns, "cluster")
```

Comment:
We can use either typed or untyped constants instead of strings like "cluster" cuz it's easy to make a typo and compiler cannot help in such case.

Reply:
totally agree

Reply:
If you still really need column names for some reason, them should be kept as constants in only one package which is responsible for the specific functionality.